src/ucp/core/ucp_request.c

```diff
@@ -309,28 +309,28 @@ ucs_status_t
 ucp_request_send_start(ucp_request_t *req, ssize_t max_short,
                        size_t zcopy_thresh, size_t zcopy_max, size_t dt_count,
                        const ucp_ep_msg_config_t* msg_config,
-                       const ucp_proto_t *proto)
+                       const ucp_proto_t *proto, int *multi_send)
```

Comment:
just noticed, we should probably move this to ucp_request.inl to make it work..

Reply:
but this is too big function for inline

Reply:
so let's revert to checking proto send func..

Reply:
ok, done